node/test/unit/tessel.js

```diff
@@ -565,7 +565,20 @@ exports['Tessel.Port.prototype'] = {
     test.done();
   },
 
-  I2C: function(test) {
+  I2CnoArgsAlwaysHasPort: function(test) {
+    test.expect(3);
+
+    var device1 = new this.port.I2C();
```

Comment:
> my question is about the options.address not the options.port

This is a pretty good indication that I need to stop working for the day. 

> Shouldn't it hit this line?

No, because `Tessel.I2C` is stubbed in these tests (this set is just for properties of a `Tessel.Port` instance.